tests/Loader/LoaderIntegrationTest.php

```diff
@@ -591,7 +594,7 @@ public function testTemplatesAreBuildBeforeUsage()
         );
 
         $actual = $this->loader->loadData([
-            \Nelmio\Alice\Entity\DummyWithConstructorAndCallable::class => [
```

Comment:
keeping a FQCN is better here as it is way more easier to follow

Reply:
Just this case or in general? I think it's better to use expressive class names, rather then depend on namespace, like `UserRepository` instead of `User` (entity? repository? controller?) and `DummyEntityWithConstructorAndCallable` instead of `DummyWithConstructorAndCallable` (dummy something?).

But I can remove this file from change.

Reply:
I agree with the practice in general, but for this case I find it more harmful than anything. It is easier to follow the FQCN (as you don't have to look up for the use statement) and a lot of classes have similar names and using use statements may require aliases. As it's only for those tests, I don't really want to bother with the names.

Reply:
Allright then.

Reply:
that's specific to that file only for those fixtures though :)

Reply:
So this whole file should be reverted or only this single case?